src/v/cluster/types.h

```diff
@@ -421,14 +421,14 @@ struct fetch_tx_reply
     model::producer_identity pid;
     model::producer_identity last_pid;
     model::tx_seq tx_seq;
-    std::chrono::milliseconds timeout_ms;
+    std::chrono::milliseconds timeout_ms{};
     tx_status status;
     std::vector<tx_partition> partitions;
     std::vector<tx_group> groups;
 
-    fetch_tx_reply() noexcept = default;
+    explicit fetch_tx_reply() = default;
```

Comment:
why this change?

Reply:
This was unintentional.. I was playing around with various constructor types to understand how initialization works, forgot to undo.